sxt/curve_g1/operation/compression.t.cc

```diff
@@ -0,0 +1,101 @@
+/** Proofs GPU - Space and Time's cryptographic proof algorithms on the CPU and GPU.
```

Comment:
Please add a test for basic usage

Take two G1 elements e1, e2 that are not equal. Compress to get ce1, ce2. Verify ce1 != ce2.

Reply:
Good idea. Added.